workflows/rnaseq/rnaseq-standard-fastq.wdl

```diff
@@ -0,0 +1,123 @@
+## # RNA-Seq Standard
+##
+## This WDL workflow runs the STAR RNA-seq alignment workflow for St. Jude Cloud.
+## The workflow takes an input BAM file and splits it into fastq files for each read in the pair. 
+## The read pairs are then passed through STAR alignment to generate a BAM file. The BAM is run
+## through several QC steps including FastQC and Qualimap. Quantification is done using htseq-count. 
+## A final QC report is produced by MultiQC to generate a combined overview of the QC results
+## for the sample.
+##
+## ## LICENSING
+##
+## #### MIT License
+##
+## Copyright 2019 St. Jude Children's Research Hospital
+##
+## Permission is hereby granted, free of charge, to any person obtaining a copy of this
+## software and associated documentation files (the "Software"), to deal in the Software
+## without restriction, including without limitation the rights to use, copy, modify, merge,
+## publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons
+## to whom the Software is furnished to do so, subject to the following conditions:
+##
+## The above copyright notice and this permission notice shall be included in all copies or
+## substantial portions of the Software.
+##
+## THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+## BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+## NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+## DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+## OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+version 1.0
+
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/fq.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/star.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/picard.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/ngsderive.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/htseq.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/samtools.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/util.wdl"
+import "https://raw.githubusercontent.com/stjudecloud/workflows/master/tools/deeptools.wdl"
+
+workflow rnaseq_standard {
+    input {
+        File gencode_gtf
+        Array[File] read_one_fastqs
+        Array[File] read_two_fastqs
+        File stardb_tar_gz
+        String strandedness = ""
+        String output_prefix = "sample"
```

Comment:
Should this basename a fastq? Not a big fan of using `sample`, but a basename will likely be junk... Maybe it should be a required input?

Reply:
I considered that. I think it should probably just be a required parameter. The extension isn't necessarily consistent.